### PR TITLE
(#25) [Fixed]: Code cleaned, PEP8 fies, improvements in settings

### LINKED ISSUE #25: Basic improvements
The project needs basic improvements before start the other tickets.

**Not to use  final urls**

image

It's better to use _reverse_lazy_

**Replace some  "__unicode__"  by "__str__"

image

**Check if inheritances are being used correctly**

For example in this case, to check if it's better to use "form_valid".

image

**Check settings**

Check the default settings. For example, DEBUG must be False by default. Use _os.environ.get_

image

### PR DESCRIPTION
**Pull request by francescarpi**
Fix #25 

## Description

This PR clean some improvable code, fixes some PEP8 mistakes and let us to use an environment variable for enable / disable django debug.

## Code formating

The code has to be well formatted following the formatting rules. For example in Python the code has to follow the PEP8.

### CODE REVIEW

**Review by XaviTorello — APPROVED** [review]
Looks fine to me, thanks @francescarpi 🍻

File: jobs/settings.py [hunk 1]
```diff
@@ -1,11 +1,22 @@
 import os
 from datetime import timedelta
 
+
+# Force to cast strings to bool
+def str2bool(value: str, default: bool = False) -> bool:
```
Comment (fullonic) [inline jobs/settings.py:6]: Hi, there is a function in the stdlib that does this convertion, `strtobool()`. It's checks for all this four 
possibilities and even for capital "T" of "F".
We would end up with something like:
 ```python
from distutils.util import strtobool

DEBUG = strtobool(os.environ.get('APF_DEBUG', False))  # returns 0 or 1
```
Does the same but a bit cleaner

Comment (francescarpi) [reply jobs/settings.py:6]: Nice fullonic! 

I'm gonna create a new ticket to improve it.

Thanks!

Comment (francescarpi) [reply jobs/settings.py:6]: Hi @fullonic, 

you can take this issue #42 and send me a pull request.

Thank you!

Comment (fullonic) [reply jobs/settings.py:6]: Great. I would make the change and create the PR. Thanks :)

**Pull request merged**